applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -269,8 +278,12 @@ static void audio_datapath_drift_compensation(uint32_t frame_start_ts)
 			return;
 		}
 
+		ctrl_blk.drift_comp.ctr = 0;
+
 		int32_t err_us = err_us_calculate(ctrl_blk.previous_sdu_ref_us, frame_start_ts);
 
+		/* Use asymptotic to get smaller corrections */
+		err_us /= 2;
```

Comment:
This is effectively the proportional part of a P-regulator. 
This is usually done by multiplication.
```suggestion
		err_us = err_us * REGULATOR_K_P;
```
Then `#define REGULATOR_K_P (0.5)`

Reply:
Wouldn't you then require a float operation, which takes more time?

Reply:
#define REGULATOR_DIV_FACTOR 2